course/assignments/reproducible-research-assignment.md

```diff
@@ -0,0 +1,21 @@
+## Reproducible Research Plan:
+
+Based on the [Data Reuse Plan](https://github.com/mozillascience/working-open-workshop/blob/gh-pages/handouts/data_reuse_plan_template.md) by Mozilla WOW.  
+
+Create a README.Rmd that describes the data or software package. It can be pretty sparse for now but you will populate it as you create your project. It must have the following sections (If a section does not apply to your project, remove it and when you submit your assignment, include a Notes.txt explaining why it is not applicable)
+
+1. Project Description: What is your project about? What is the goal? Why are you doing it? How does this data relate to your project?
+    - Data Source: If you're using an extrnal data set, provide information on where it came from:
+    ```
+    e.g. Forstmann BU, et al. (2014) Data from: Multi-modal ultra-high resolution structural 7-Tesla MRI data repository. Dryad Digital Repository. (http://dx.doi.org/10.5061/dryad.fb41s)
+    ```
+2. Who We Are: Information about the people/organizations responsible for this project.
+    ```
+    e.g. Jane Doe, University of Toronto, Dept of Ecology and Evolutionary Biology
+    ```     
+3. Installation and Usage Instructions for Software: General installation instructions for software you will use can go here. Re: R, necessary R packages etc.
+4. File/Folder Naming and Structure: Explain what files will be found in each folder and naming conventions used. Eventually, a listing of folders and files included in the data set. This can change as you develop your project and find what works best, but you must outline something for this assignment.
+5. Abbreviations and Column Names: describe abbreviations, column names, etc relevant to your project. If you can think of any now, fill them in, else, populate as you work on your project.
```

Comment:
Not sure either, what do you have in mind for abbreviations @linamnt ? Do you mean abbreviated variable names? For column names, can't this be inculded as metadata in the data frame objects? If so, is that preferable to having it in a readme? I am not sure which is best, I just downloaded the USDA nutrition database, and that one comes with a README pdf listing all column names and what they mean, so maybe the README is the preferred place.